types/schematypes.d.ts

```diff
@@ -216,6 +216,9 @@ declare module 'mongoose' {
     /** Attaches a getter for all instances of this schema type. */
     static get(getter: (value: any) => any): void;
 
+    /** Array containing default setters for all instances of this SchemaType */
+    static setters: Function[];
```

Comment:
```suggestion
    static setters: ((val?: unknown) => unknown)[];
```

i think it is better with known arguments, unless there are more arguments?